f5/file.go

```diff
@@ -36,5 +36,10 @@ func (f *Device) UploadFile(filename string, data []byte) (error) {
 		s := buf.String()
 		return errors.New("Unable to process request, returned status: " + response.Status + " " + s)
 	}
```

Comment:
There's a bug in the f5 where uploaded files cannot be used by the daemon that processes certificates and keys due to the two interfaces using different accounts, this makes the file readable for all users.